src/api/tasks.js

```diff
@@ -1,14 +1,14 @@
 
 import Q from "q";
 import logger from "winston";
-import { Task } from "../model/tasks";
-import { Transaction } from "../model/transactions";
-import { AutoRetry } from "../model/autoRetry";
+import { TaskModelAPI } from "../model/tasks";
+import { TransactionModelAPI } from "../model/transactions";
+import { AutoRetryModelAPI } from "../model/autoRetry";
 import * as Channels from "../model/channels";
```

Comment:
ok cool i think that is fine to keep as is then, I see its being used to determine if a channel is active as well and seems more readable in the way that is now